src/coreclr/gc/gcinterface.h

```diff
@@ -1013,6 +1013,8 @@ class IGCHeap {
     virtual FinalizerWorkItem* GetExtraWorkForFinalization() PURE_VIRTUAL
 
     virtual uint64_t GetGenerationBudget(int generation) PURE_VIRTUAL
+
+    virtual uint64_t GetLOHThreshold() PURE_VIRTUAL
```

Comment:
Would `size_t` make more sense as the return type? This cannot be more than `size_t`. It would save the casts in all places that it is called.

I see that you have matched `GetGenerationBudget`, but the return type of `GetGenerationBudget` looks like an oversight. There are many address-space sized values in gcinterface.h that use `size_t`.